### ISSUE TITLE
Programmatic name is not given for all checkboxes present under get started with accessibility feature section: A11y_Visual Studio Code Client_get started with accessibility feature_Devtools

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#241175]
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver**
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

## GitHub Tags:
#A11yeDAD; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG4.1.2; #Devtools; #FTP; #Win32;

## Environment and OS details:
Visual Studio Code version 1.97.2
Microsoft Windows 11 Enterprise (24H2)
OS Build 26100.2605

## Repro Steps:
1.Open Visual studio code
2. TAB till Help menu item 
3. TAB till Get started with accessibility feature and press ENTER key.
5. TAB till checkbox present under Get started with accessibility feature section
6. Observe that programmatic name is not given for all checkboxes present under get started with accessibility feature section.

## Actual:
Programmatic name is not given for all checkboxes present under get started with accessibility feature section.

## Expected:
Associate label should be provided as programmatic name for all check box present under get started with accessibility feature section.

## User Impact:
Screen reader user will face difficulty, as screen reader will announce as checkbox only. 

## Attachment:

![Image](https://github.com/user-attachments/assets/905caddc-86a2-4ea2-82a1-3a12a72c17cd)

**Comment by kapilvaishna** [comment]
Issue is not repro in below environment
So closing the bug.

Visual studio code insider
Version 1.99.0

![Image](https://github.com/user-attachments/assets/e7b49999-9846-4bb4-a664-1ae83e17ca47)
